frontend/app/NearlineUnclog.jsx

```diff
@@ -0,0 +1,139 @@
+                    let data;
+                    if(this.state.chartMode===NearlineControlsBanner.CHART_MODE_COUNT){
+                        data = [datapoint.count];
+                    } else if(this.state.chartMode===NearlineControlsBanner.CHART_MODE_SIZE){
+                        data = [datapoint.size];
+                    } else {
+                        throw "Invalid mode for component, should be count or size."
+                    }
+                    return {data: data,label:datapoint.label, backgroundColor: NearlineUnclog.colourValues[idx*2], hoverBackgroundColor: NearlineUnclog.hoverColourValues[idx*2] }
+                }) : [],
+                labels: ["Files"]
+            }}
+           height={50}
+           options={{scales: {
+                   yAxes: [{stacked: true}],
+                   xAxes: [{
+                       stacked: true,
+                       labelString: this.state.chartMode===NearlineControlsBanner.CHART_MODE_SIZE ? "Total size" : "File count",
+                       ticks: {
+                           callback: (value,index,series)=>
+                               this.state.chartMode===NearlineControlsBanner.CHART_MODE_SIZE ? Math.floor(value/1099511627776) + "Tib" : value
+                       }
+                   }]
+
+               },
+               tooltips: {
+                   callbacks: {
+                       label: (tooltipItem,data)=>{
+                           let xLabel=data.datasets[tooltipItem.datasetIndex].label;
+                           let yLabel=data.datasets[tooltipItem.datasetIndex].data;
+
+                           console.log(tooltipItem);
+                           console.log(data);
+                           console.log(xLabel, yLabel);
+
+                           if(this.state.chartMode===NearlineControlsBanner.CHART_MODE_SIZE) {
+                               try {
+                                   const result = BytesFormatterImplementation.getValueAndSuffix(yLabel);
+                                   yLabel = result[0] + result[1];
+                                   return xLabel + ": " + yLabel;
+                               } catch (err) {
+                                   console.error(err);
+                                   return xLabel + ": " + yLabel;
+                               }
+                           } else {
+                               return xLabel + ": " + yLabel;
+                           }
+                       }
+                   }
+               }
+           }}
+           getElementAtEvent = {elems=>{
+               const bar = elems[0];
+               console.log("You clicked on bar number ", bar._datasetIndex, bar._model.datasetLabel);
+               this.setState({selectedProjectStatus: bar._model.datasetLabel, showProjectsList: true});
+           }}
+
+            />
+            <div>
+                <br />
```

Comment:
would be much better to set a margin or padding on the containing div than rely on a br here

Reply:
I tried that but could not get it to work because it did not like the - character.

Reply:
The br tag has been removed.